bodhi/server/config.py

```diff
@@ -412,7 +412,13 @@ class BodhiConfig(dict):
             'value': '%s has been submitted as an update to %s. %s',
             'validator': six.text_type},
         'greenwave_api_url': {
-            'value': 'https://greenwave.fedoraproject.org/api/v1.0',
+            'value': 'https://greenwave-web-greenwave.app.os.fedoraproject.org/api/v1.0',
```

Comment:
Normally I wouldn't want to change a default setting without raising the X version of the release, but in this case it does appear that the default does not exist:

```
$ host greenwave.fedoraproject.org
Host greenwave.fedoraproject.org not found: 3(NXDOMAIN)
```